service/dap/server.go

```diff
@@ -71,6 +87,8 @@ type Server struct {
 	// args tracks special settings for handling debug session requests.
 	args launchAttachArgs
 
+	// mu synchronizes access to objects set on start-up (from run goroutine)
+	// and stopped on teardown (from main goroutine)
```

Comment:
fyi - Since there is no mutex annotation in go, frequently used convention is to lay out the fields protected by a mutex after the mutex. For example, once you we start protecting debugger or conn with mu, place them after mu.

Reply:
done